ip.go

```diff
@@ -57,9 +57,17 @@ type IpAddressCommon struct { //nolint:golint
 	Project       Href        `json:"project"`
 	Global        bool        `json:"global_ip"`
 	Tags          []string    `json:"tags,omitempty"`
+	ParentBlock   ParentBlock `json:"parent_block,omitempty"`
```

Comment:
Was your experience that this field was not present in the response when not included?
Or was it that `parent_block` is always returned, but only contains `href`?

In the former case, this should be a `*ParentBlock` to signify to the user that the parent block was not reported, distinct from being reported but empty (or only containing Href).

Reply:
Not present when not included, so, yes, this would be better as a pointer. Will update.

Reply:
Works better that way. When not included, you get a nil.